plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -1010,6 +1040,32 @@ def get_want_authentication_policy_server(self, auth_policy_server):
                         "ipAddress": ip_address
                     })
 
+                    fqdn = ise_credential.get("fqdn")
+                    if not fqdn:
+                        if not auth_server_exists:
+                            self.msg = "The required parameter 'fqdn' is missing when 'server_type' is 'ISE'."
+                            self.status = "failed"
+                            return self
+
+                        response = self.dnac._exec(
+                            family="system_settings",
+                            function='get_authentication_and_policy_servers',
+                            params={"is_ise_enabled": True}
+                        )
+                        response = response.get("response")
+                        if response is None:
+                            self.msg = (
+                                "Failed to retrieve the information from the API 'get_authentication_and_policy_servers' of {0}."
+                                .format(ip_address)
+                            )
+                            self.status = "failed"
+                            return self
+
+                        cisco_ise_dtos = response[0].get("ciscoIseDtos")
+                        fqdn = self.get_primary_ise_fqdn(cisco_ise_dtos, ip_address)
```

Comment:
self.log(
        "Retrieved FQDN for primary ISE node with IP '{ip}': {fqdn}"
        .format(ip=ip_address, fqdn=fqdn),
        "INFO"
    )